src/components/UserManagement/UserTableData.jsx

```diff
@@ -90,6 +99,7 @@ const UserTableData = React.memo(props => {
         <button
           type="button"
           className={`btn btn-outline-${props.isActive ? 'warning' : 'success'} btn-sm`}
+          disabled={isOwner? false : true}
```

Comment:
```suggestion
```
Not related to creating users

Reply:
> Not related to creating users

I have updated the code. But we need to disable these.

Please check the feature description:
If the owner gives users like (Owners, Administrators, Core Team, Manager, Mentors, Volunteers) “create new users” access in “permission management, then this individual should see “user management” button, and be able to create new users, while at the same time, cannot use “delete”, “reset password”, “set final day”, “pause”, “request time off” these modification functionalities.

If the owner revokes anyone’s “create new users” access in “permission management”, then the person cannot see “user management” button.